onnxruntime/contrib_ops/cpu/quantization/gather_block_quantized.cc

```diff
@@ -232,7 +287,7 @@ template <typename T1, typename Tind>
 Status GatherBlockQuantized<T1, Tind>::Compute(OpKernelContext* context) const {
   Prepare p;
   ORT_RETURN_IF_ERROR(PrepareForCompute(context, p));
-  auto components = (std::is_same_v<T1, uint8_t>) ? 2 : 1;
+  int64_t components = std::is_same_v<T1, uint8_t> ? (8 / static_cast<int>(bits_)) : 1;
```

Comment:
Do we need `static_cast<int>` since `bits_` is already `int64_t`

Reply:
It fixes a compiler warning.  8 is int, bits is int64_t, the warning is that need to match type for /